src/main/java/com/tomtom/services/configuration/dto/ParameterListDTO.java

```diff
@@ -35,8 +35,8 @@
      * class only needs to implement a method to check a single list element.
      */
     @Override
-    public void validateOne(@Nonnull final ParameterDTO elm) {
-        validator().checkNotNullAndValidate(false, "parameter", elm);
+    public void validateOne(@Nonnull final ParameterDTO t) {
```

Comment:
This was an existing IntelliJ warning, it overrides a function that calls it "t".